config/config.go

```diff
@@ -49,7 +49,7 @@ type Config struct {
 		ServiceRelativePath   string
 		InstanceRelativePath  string
 		ExecutionRelativePath string
-		WorkflowRelativePath  string
+		ProcessRelativePath   string
```

Comment:
Remove RelativePath as engine dosen't need it them and they are not used anymore.

Reply:
This is totally needed in order to create and access the database